src/TaskList.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+interface Task {
+  text: string;
+  completed: boolean;
+}
+
+interface TaskListProps {
+  tasks: Task[];
+  handleCompleted: (index:number) => void
+}
+
+function TaskList({ tasks, handleCompleted }: TaskListProps) {
+
+
+  console.log("Tareas en TaskList:", tasks);
+
+  return (
+    <section className="tasks">
+      <ul className="task-list">
+        {tasks.map((item, index) => (
+        <li className={item.completed ? 'complete' : 'noCompleted'}onClick={() => handleCompleted(index)} >
```

Comment:
Add a space before onClick.

We don't really need a `noCompleted` class with a `text-decoration: none`. If we remove that class, text won't have the line. It could be something like

```typescript
<li className={item.completed ? 'complete': '' ...>
```